backend/controllers/webhook.js

```diff
@@ -42,6 +44,26 @@ router.post('/', (req, res) => {
       console.log(`Meeting ${meetingName} started at ${timeStarted}`);
       return res.status(201).json({ message: 'Meeting has started.' });
     }
+    case WEBHOOK_PARTICIPANT_JOINED:{
+         console.log('participant joined', req.body.payload.object.participant);
+        const participant_info = JSON.stringify({
+          account_id: req.body.payload.account_id,
+          user_name: req.body.payload.object.participant.user_name,
+          phone_number: req.body.payload.object.participant.phone_number
+            ? req.body.payload.object.participant.phone_number
+            : '',
+        });
```

Comment:
Can you add a console log similar to the following: `${participantId} - ${participantName} joined ${meetingName} at ${participantJoinedTime}`